tests/testthat/test-object_usage_linter.R

```diff
@@ -1,118 +1,150 @@
-context("object_usage_linter")
 test_that("returns the correct linting", {
   expect_lint("blah",
-    NULL,
-    object_usage_linter)
+              NULL,
+              object_usage_linter)
 
   expect_lint(
-"function() {
-  a <- 1
-  a
-}",
+    trim_some("
```

Comment:
Usage of trim_some() greatly improves readability of these tests.
Their behaviour is unchanged.